certvfy/README.md

```diff
@@ -0,0 +1,26 @@
+# wolfSSL Cert Verification Examples
+
+This directory contains:
+
+A simple example of using the low level wolfSSL APIs to verify a certificate
+in a standalone manner, separate from an SSL/TLS connection.
+
+## Compiling and Running the Example
+
+```
+$ cd wolfssl
+$ ./autogen.sh                # If downloaded from github
+$ ./configure '--enable-cryptonly' '--enable-singlethreaded' '--enable-opensslextra'
```

Comment:
Is opensslextra really required?

Reply:
wc_CheckCertSigPubKey is only available when OPENSSL_EXTRA is defined.